digitalocean/participate.yaml

```diff
@@ -0,0 +1,32 @@
+# https://www.digitalocean.com/docs/app-platform/references/app-specification-reference/
+domains:
+- domain: staging.participate.whatwg.org
+  type: PRIMARY
+  zone: whatwg.org
+name: participate
+region: nyc
+services:
+- dockerfile_path: Dockerfile
+  envs:
+  - key: PORT
+    scope: RUN_TIME
+    value: '8080'
+  - key: PRIVATE_CONFIG_JSON
+    scope: RUN_TIME
+    type: SECRET
+    value: __PRIVATE_CONFIG_JSON__
+  - key: VERSION
+    scope: RUN_TIME
+    value: ${_self.COMMIT_HASH}
+  github:
+    branch: main
+    deploy_on_push: true
+    repo: foolip/participate.whatwg.org
+  health_check:
+    http_path: /
+  http_port: 8080
+  instance_count: 1
+  instance_size_slug: basic-xxs
+  name: participate
+  routes:
```

Comment:
What does this do?

Reply:
It's the routes under participate.whatwg.org which should be handled. I don't think there's a default and that I had to add this.